Source/JavaScriptCore/runtime/Structure.h

```diff
@@ -174,7 +174,12 @@ class Structure : public JSCell {
     typedef JSCell Base;
     static constexpr unsigned StructureFlags = Base::StructureFlags | StructureIsImmortal;
     static constexpr uint8_t numberOfLowerTierCells = 0;
-    
+
+#if ENABLE(STRUCTURE_ID_WITH_SHIFT)
+    static constexpr size_t atomSize = 32;
+#endif
+    static_assert(JSCell::atomSize >= MarkedBlock::atomSize);
```

Comment:
Does Structure.h already #include MarkedBlock.h indirectly?  I suggested putting this static_assert in a constructor or maybe just at the top of the .cpp file because I wasn't expecting Structure.h to #include MarkedBlock.h.

Reply:
Yes, we have this.